### PR TITLE
Start call fixes

### PR DESCRIPTION
**Pull request by aleksMooncascade**

### CODE REVIEW

File: widgetssdk/build.gradle [hunk 1]
```diff
@@ -40,7 +40,7 @@ ext {
     appCompatVersion = '1.2.0'
     materialVersion = '1.2.1'
     constraintLayoutVersion = '2.0.4'
-    gliaSdkVersion = '0.17.0'
+    gliaSdkVersion = '0.19.0'
```
Comment (deniss-mc) [inline widgetssdk/build.gradle:43]: Good 👍

File: widgetssdk/src/main/java/com/glia/widgets/call/CallController.java [hunk 2]
```diff
@@ -601,8 +647,14 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
         emitViewState(callState.engagementStarted(
                 engagement.getOperator().getName(),
                 operatorProfileImgUrl,
-                Utils.toMmSs(0))
+                String.valueOf(0))
```
Comment (deniss-mc) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:650]: just "0"?

File: widgetssdk/src/main/java/com/glia/widgets/call/CallView.java [hunk 3]
```diff
@@ -311,23 +312,27 @@ public void emitState(CallState callState) {
                             appBar.setTitle(resources.getString(R.string.call_audio_app_bar_title));
                         }
                     }
-                    operatorStatusView.isRippleAnimationShowing(callState.isCallNotOngoing());
+                    operatorStatusView.isRippleAnimationShowing(
+                            callState.isCallNotOngoing() ||
+                                    callState.isCallOngoig()
```
Comment (deniss-mc) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallView.java:317]: typo

Comment (deniss-mc) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallView.java:317]: is there btw any state where call is not ongoing and ongoing at the same time?

Comment (aleksMooncascade) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallView.java:317]: It is || (OR), meaning when there is no media states pretty much..

File: widgetssdk/src/main/java/com/glia/widgets/call/CallController.java (hunk 2 above)
Comment (aleksMooncascade) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:650]: Yeah its this view: https://www.figma.com/file/x5BLxm2UWpm3TuerdqmYi8/Glia-Mobile-UI-Components?node-id=263%3A1889

**Review by deniss-mc — APPROVED** [review]

**Pull request merged**